test/compile/Model.test.ts

```diff
@@ -60,9 +60,12 @@ describe('Model', function() {
         mark: "point",
         encoding: {
           x: {field:'a', type: "quantitative"}
+        },
+        config: {
+          numberFormat: 'foo'
```

Comment:
Although `’foo’` is fine placing here for a purpose of unit testing, I prefer to a value that is a valid in real life setting as well.